string name
duration time
---
bool success